src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -174,7 +175,7 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
-    const isCurrentPermTypeDelegated = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
+    const delegatedPermissions = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
```

Comment:
I prefer the `is_______` naming convention over the simpler `delegatedPermissions` variable name because the `is` part implies the variable is a boolean, any reason why it's being changed now?

Reply:
I changed it from the is__ based on this comment: https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/16#discussion_r663962381